src/minimizer_mapper.hpp

```diff
@@ -94,6 +94,9 @@ class MinimizerMapper : public AlignerClient {
     /// of total score
     double minimizer_score_fraction = 0.9;
 
+    /// Take at maximum number of unique non-overlapping minimizers
```

Comment:
```suggestion
    /// Maximum number of distinct minimizers to take
```
"Take at maximum" doesn't make sense to me, and they aren't non-overlapping unless the flag is set later. And we should make it clear we mean unique as in different form each other, not as in single-hit.